ssg/constants.py

```diff
@@ -242,6 +242,9 @@
     anssi=Reference(
         id="anssi", name="ANSSI", url=anssi_ns,
         regex_with_groups=r"BP28\(R(\d+)\)"),
+    ccn=Reference(
+        id="ccn", name="CCN", url="",
+        regex_with_groups=r"A.(\d+).SEC-RHEL(\d+)"),
```

Comment:
This fine for now, but if other products (SUSE, etc) need a CCN reference we will need to change this.